src/assertions/OrderAssertions.ts

```diff
@@ -182,6 +182,21 @@ export class OrderAssertions {
     );
   }
 
+  /* ============ Private Helpers =============== */
```

Comment:
Could we move `assertRequiredComponentsAndAmounts` below into the private methods section?